frontend/views/containers/chatroom/ChatMain.vue

```diff
@@ -857,8 +833,6 @@ export default ({
         events = await sbp('chelonia/out/eventsAfter', chatRoomID, sinceHeight, latestHeight - sinceHeight + 1, undefined, { stream: false })
       }
 
-      if (!this.checkEventSourceConsistency(chatRoomID)) return
```

Comment:
I'm realizing that in `async` methods specifically, we still need these `checkEventSourceConsistency` type checks in some places (i.e. similar to how we had it before, either before or after a call to `await`) because these `await` calls represent a time-split where the chatroom couldn't been changed .

Reply:
have created `this.chatroomHasSwitchedFrom(chatroomID)` method and added it to a few places as well as `scrollToMessage`